README.md

```diff
@@ -26,7 +27,9 @@ Demonstrate calling the method with an argument of "young prince".
 
 Write your code here:
 ```ruby
-# code here
+def offerRose
+  @person = person
+  puts "would you take this rose and help out an old beggar" + {@person}
```

Comment:
String interpolation has to follow the syntax in my example above, e.g. we do it inside the quotes for the string.